src/bundle/Resources/public/scss/ui/modules/universal-discovery/_grid.scss

```diff
@@ -1,5 +1,6 @@
 .c-grid {
     height: 100%;
+    overflow: hidden !important;
```

Comment:
Instead of using `!important` I think it would be better to remove the original styling in `m-ud`